jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/MainActivity.java

```diff
@@ -1,13 +1,33 @@
 package com.e_conomic.jonfirstapp;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
 
 public class MainActivity extends AppCompatActivity {
 
+    // Key for the intent extra holding the message.
+    public final static String EXTRA_MESSAGE = "com.e-conomic.jonfirstapp.MESSAGE";
```

Comment:
We do't have these conventions because our app is not listening to other apps' intents. We just try to make sure they're unique within the app.